frontend/src/components/App/RouteSwitcher.tsx

```diff
@@ -18,7 +18,7 @@ export default function RouteSwitcher() {
           route.name === 'OidcAuth' ? (
             <Route
               path={route.path}
-              component={() => (
+              render={() => (
```

Comment:
I need to understand better why we need this. AFAIU, if we don't unmount when a route is changed, it means we'll increase the memory footprint and the whole UI may become much slower after some use.

Reply:
any state change will cause the component to get re rendered in case of a component prop a new component is rendered each time rather than rendering with some parts changed
https://reactrouter.com/web/api/Route/render-func